english/chuck-norris/chucknorris.md

```diff
@@ -0,0 +1,6 @@
+You pull out your cell phone and text Chuck Norris to ask what he would do.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.